pages/_app.tsx

```diff
@@ -1,12 +1,9 @@
 import React from 'react';
 import '../styles/globals.scss';
 import type { AppProps } from 'next/app';
-import Layout from '../components/common/layout';
 
 function MyApp({ Component, pageProps }: AppProps) {
-    return <Layout pageName={Component.name}>
```

Comment:
Looks like component names are minified during the build process resulting in browser titles like

![image](https://user-images.githubusercontent.com/667983/187047809-dde60ed5-9dcd-47bf-aaf9-2815dd859086.png)

So the [comment ](https://github.com/jarekb84/my/pull/35#discussion_r956618947)I made in the first PR about the property serving a different intent...ended up biting me pretty quickly.